nodes/parachain/src/command.rs

```diff
@@ -86,30 +86,30 @@ fn runtime(id: &str) -> Runtime {
 
 fn load_spec(id: &str) -> std::result::Result<Box<dyn ChainSpec>, String> {
 	log::info!("Load spec id: {}", id);
-	let polimec_chain_spec = include_bytes!("../../../chain-specs/polimec-raw-chain-spec.json").to_vec();
+	let polimec_chain_spec = include_bytes!("../../../chain-specs/polkadot/polimec-raw-chain-spec.json").to_vec();
 	let rolimec_chain_spec = include_bytes!("../../../chain-specs/rococo/rawlimec.json").to_vec();
 
 	Ok(match id {
-		// Base runtime
-		"polimec" => Box::new(chain_spec::polimec::ChainSpec::from_json_bytes(polimec_chain_spec)?),
-		"rolimec" => Box::new(chain_spec::polimec::ChainSpec::from_json_bytes(rolimec_chain_spec)?),
-		"politest" => Box::new(chain_spec::politest::get_live_chain_spec()),
-		"polimec-local" => Box::new(chain_spec::polimec::get_local_chain_spec()),
-		"rolimec-local" => Box::new(chain_spec::polimec::get_rococo_chain_spec()),
-		"politest-local" => Box::new(chain_spec::politest::get_local_chain_spec()),
-		// -- Fallback (generic chainspec)
+		// Original live Polimec runtimes
+		"polimec" => Box::new(chain_spec::common::ChainSpec::from_json_bytes(polimec_chain_spec)?),
```

Comment:
Can we please add the raw chainspec in the node so we can start a collator by using something like `--chain polimec-paseo`?